frontend/device/gesturedetector.lua

```diff
@@ -1067,6 +1067,14 @@ function Contact:handleTwoFingerPan(buddy_contact)
             ges = "two_finger_pan",
             -- Use midpoint of tstart and rstart as swipe start point
             pos = tstart_pos:midpoint(rstart_pos),
+            -- Use the the sum of both contacts' travel for the distance:
+            -- there's going to be some overlap (on sane symmetric gestures,
+            -- it's going to be almost the double of a similar one contact swipe),
+            -- but it's easier to deal with than trying to compute the longest path *across* the two fingers,
+            -- e.g., in [--s1----s2--e1---e2-], s1 to e2, which is NOT max(s1 to e1, s2 to e2)
+            -- (that's just the longest distance of *one* finger among the two).
+            -- Code that consumes such gestures usually corrects for that
+            -- (c.f., the very unfortunately named FRONTLIGHT_SENSITIVITY_DECREASE global constant).
```

Comment:
Don't you prefer correcting these bits of code (which must not be legion), instead of continue carrying this odd stuff ?

Reply:
That's the plan today ;).